scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RunProcessor.java

```diff
@@ -127,25 +126,32 @@ public static ObjectMapper createObjectMapper() {
     return mapper;
   }
 
+  /** Attempt to parse naively, and force parsing using UTF-8 if that doesn't work */
   public static Optional<Document> parseXml(File file) {
     try {
-      DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+      return Optional.of(DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(file));
+    } catch (ParserConfigurationException | IOException | SAXException e) {
+      log.error("Not really a UTF-16 parsing exception, forcing UTF-8 parsing...", e);
```

Comment:
- Which exception is actually thrown when this happens? Only that exception should be handled in this way. Fail fast for the others by catching in a separate block (which you already have below catching `Exception`).
- Change to warning level. It's not an error at this point since we're attempting to handle it.